### ISSUE TITLE
Problem switching identity

### ISSUE THREAD

**Issue by jvromans** [keybase/keybase-issues#2829]
Using the app on Linux, there is no way to log off and log on again into a different id.

**Comment by strib** [comment]
Not sure about the app, but you can do this on the command line with `keybase logout` and `keybase login <other_username>`.

**Comment by jvromans** [comment]
Yes, but it is a bit weird to quit the app, logout/login on the command line, and then restart the app just to switch user.
Although I must admit that switching user will not be very common, loggin out definitely is!
